bin/medic.js

```diff
@@ -39,9 +65,109 @@ var argv = optimist
     .argv;
 
 // helpers
+function killTasks(taskNames) {
+
+    if (!taskNames || taskNames.length < 1) {
+        console.warn("no tasks to kill");
```

Comment:
No tasks seems to equal success case.  If truly a usage error, throw, but otherwise this seems unnecessarily chatty.